node/src/main/scala/com/wavesplatform/network/NetworkServer.scala

```diff
@@ -202,37 +206,47 @@ object NetworkServer extends ScorexLogging {
         }
       )
 
-    val connectTask = workerGroup.scheduleWithFixedDelay(1.second, 5.seconds) {
-      import scala.collection.JavaConverters._
-      val outgoing = outgoingChannels.keySet.iterator().asScala.toVector
+    def scheduleConnectTask(): Unit = {
```

Comment:
Please avoid `return` at all costs!
```suggestion
    def scheduleConnectTask(): Unit = if (!shutdownInitiated) {
```